tests/tests/optionsTests.robot

```diff
@@ -0,0 +1,95 @@
+
+
+*** Test Cases ***
+Find and open Enable Debug Output
+    [Documentation]    Click Options, then toggle on/off Enable Debug output button and verify if Reload Badge appears/disappears
+    [Tags]    options
+
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${OptionsList}
+
+    Click Element    ${EnableDebugSwitch}
+    Wait Until Element Is Visible    ${ReloadBadge}
+
+    Click Element    ${EnableDebugSwitch}
+    Wait Until Element Is Not Visible    ${ReloadBadge}
+
+Find and open Exclude temporary files
+    [Documentation]    Click Options, then toggle on/off Exclude temporary files switch button and verify if Reload Badge appears/disappears
+    [Tags]    options
+
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${OptionsList}
+
+    Click Element    ${ExcludeTemporaryFilesSwitch}
+    Wait Until Element Is Visible    ${ReloadBadge}
+
+    Click Element    ${ExcludeTemporaryFilesSwitch}
+    Wait Until Element Is Not Visible    ${ReloadBadge}
+
+Find and open Output logs to a file
+    [Documentation]    Click Options, then toggle on/off Output logs to a file button and verify if Reload Badge appears/disappears
+    [Tags]    options
+
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${OptionsList}
+
+    Click Element    ${OutputLogsSwitch}
+    Wait Until Element Is Visible    ${LogsButton}
+    Wait Until Element Is Not Visible    ${ReloadBadge}
+
+Find and open Extraction language filter
+    [Documentation]    Click Options, then Extraction language filter option, select every option and verify if Reload Badge appears/disappears
+    [Tags]    option
+
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${OptionsList}
+
+    Click Element    ${ExtractionLangFilterOption}
+
+    Click Element    ${EverythingOption}
+    Wait Until Element Is Visible    ${ReloadBadge}
+
+    Click Element    ${OnlyChosenLanguageOption}
+    Wait Until Element Is Visible    ${ReloadBadge}
+
+    Click Element    ${OnlyLanguageAgnosticOption}
+    Wait Until Element Is Visible    ${ReloadBadge}
+
+    Click Element    ${ChosenLanguageAndLangAgnosticOption}
+    Wait Until Element Is Not Visible    ${ReloadBadge}
+
+Find and open Collision resolution option
+    [Documentation]    Click Option, then Collision resolution, select every option and verify if Reload Badge appears/disappears
+    [Tags]    options
+
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${OptionsList}
+
+    Click Element    ${CollisionResolutionOption}
+
+    Click Element    ${OverwriteOption}
+    Wait Until Element Is Not Visible    ${ReloadBadge}
+
+    Click Element    ${RenameOption}
+    Wait Until Element Is Visible    ${ReloadBadge}
+
+    Click Element    ${RenameAllOption}
+    Wait Until Element Is Visible    ${ReloadBadge}
+
+    Click Element    ${ErrorOption}
+    Wait Until Element Is Visible    ${ReloadBadge}
```

Comment:
I think it is better to remove that spaces between options since right now it unnecessarily takes too much space.

Reply:
Done